af_ktls.c

```diff
@@ -42,6 +42,8 @@
 #define KTLS_NONCE_SIZE			8
 
 #define KTLS_DATA_PAGES			(KTLS_MAX_PAYLOAD_SIZE / PAGE_SIZE)
+// Maximum value for msg_iovlen
+#define KTLS_SGL_MAX_ENTRIES    1024
```

Comment:
That is the max iov_len allowed in linux.
https://books.google.com/books?id=ptSC4LpwGA0C&pg=PA390&lpg=PA390&dq=sendmsg+iovec+1024&source=bl&ots=Ks6CQobmPp&sig=x1l8Od6bxSuwOm3FPRk0zIh-blM&hl=en&sa=X&ved=0ahUKEwidm8uqhJ7NAhUL7mMKHTV-DXAQ6AEIIzAB#v=onepage&q=sendmsg%20iovec%201024&f=false